android/modules/ui/src/java/ti/modules/titanium/ui/widget/webview/TiUIWebView.java

```diff
@@ -768,35 +895,39 @@ public void goForward()
 	public void reload()
 	{
 		switch (reloadMethod) {
-		case DATA:
-			if (reloadData != null && reloadData instanceof TiBlob) {
-				setData((TiBlob) reloadData);
-			} else {
-				Log.d(TAG, "reloadMethod points to data but reloadData is null or of wrong type. Calling default", Log.DEBUG_MODE);
-				getWebView().reload();
-			}
-			break;
-			
-		case HTML:
-			if (reloadData == null || (reloadData instanceof HashMap<?,?>) ) {
-				setHtml(TiConvert.toString(getProxy().getProperty(TiC.PROPERTY_HTML)), (HashMap<String,Object>)reloadData);
-			} else {
-				Log.d(TAG, "reloadMethod points to html but reloadData is of wrong type. Calling default", Log.DEBUG_MODE);
-				getWebView().reload();
-			}
-			break;
-		
-		case URL:
-			if (reloadData != null && reloadData instanceof String) {
-				setUrl((String) reloadData);
-			} else {
-				Log.d(TAG, "reloadMethod points to url but reloadData is null or of wrong type. Calling default", Log.DEBUG_MODE);
+			case DATA:
```

Comment:
retain existing formatting if you're not changing anything here to avoid confusion on reviews, please.